netket/logging/hdf5_log.py

```diff
@@ -0,0 +1,243 @@
+import os
+import time
+import h5py
+import numpy as np
+from os import path as _path
+from ..logging import RuntimeLog
+from ..stats import Stats
+from ..jax import tree_ravel
+
+
```

Comment:
hide the import behind a try-catch
```suggestion
try:
    import h5py
except ImportError:
    pass
```